google-cloud-logging/src/main/java/com/google/cloud/logging/MonitoredResourceUtil.java

```diff
@@ -180,11 +180,12 @@ private static String getValue(Label label, String resourceType) {
         value = MetadataConfig.getNamespaceId();
         break;
       case NamespaceName:
+        String filePath = System.getenv("NAMESPACE");
+        if (filePath == null) {
+          filePath = "/var/run/secrets/kubernetes.io/serviceaccount/namespace";
+        }
```

Comment:
"NAMESPACE" is a very generic name it may conflict with other user's variables.
Could you rename it to refect kubernetes in the name, like K8N_NAMESPACE_FILE or KUBERNETES_NAMESPACE_FILE

Reply:
Done